instance.go

```diff
@@ -174,8 +174,10 @@ func (i *Instance) Translate(to SyntacticVersion) (*Instance, TranslationLacunas
 	lac := make(multiTranslationLacunas, 0)
 	out.LookupPath(cue.MakePath(cue.Str("lacunas"))).Decode(&lac)
 
+	raw, _ := out.LookupPath(cue.MakePath(cue.Str("result"), cue.Str("result"))).Default()
```

Comment:
this is...slightly bizarre, let's make sure to add a comment indicating why the `Default()` call is there. Especially given that this is tightly coupled to the current implementation approach for `#Translate`

Reply:
Sure! Comment added!